app/views/editions/edition_thread/_assigned_to_event.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="event panel panel-default">
   <div class="panel-heading">
     <div class="pull-right">
-      <%= format_time(event.try(:updated_at) || event.try(:at)) %>
+      <%= format_time(event.edition.updated_at) %>
```

Comment:
I've written this code with the idea that it will probably change soon and just use a built in time format.